src/DynamoCoreWpf/ViewModels/Preview/InfoBubbleViewModel.cs

```diff
@@ -153,12 +416,27 @@ public InfoBubbleViewModel(DynamoViewModel dynamoViewModel)
             ConnectingDirection = Direction.None;
             Content = string.Empty;
             DocumentationLink = null;
-            ZIndex = 3;
+            // To appear above any NodeView elements
+            ZIndex = 7; 
             InfoBubbleStyle = Style.None;
             InfoBubbleState = State.Minimized;
+
+            NodeMessages.CollectionChanged += NodeInformation_CollectionChanged;
```

Comment:
Seems we need to unsubscribe somewhere?

Reply:
agree, was this ever taken care of?

Reply:
@mjkkirschner Thanks for checking these out! @OliverEGreen was away and should be able to address these comments on a new PR